### PR TITLE
Feature/cypress skeleton

### PR DESCRIPTION
**Pull request by battan40**
### Is this a fix or a feature?
* Feature

### What is the change or fix?
* Adds Cypress testing for current functionality of the project
* Currently each line of actionable code is testing and passing
* We will need more testing as we add more features, lines, technology
* The components needed small changes like class names or `jsx` tag structure
* There were opportunities to *DRY* the code; those opportunities were implemented.

### How should this be tested?
* In one terminal tab, in the root directory of the project: 
* run `npm start`
* In a separate terminal tab, also in the root directory of the project run:
* run `npx cypress open`

### Linked Issues:
* #10

### CODE REVIEW

File: src/utilities/utils.js [hunk 1]
```diff
@@ -2,20 +2,32 @@ const utils = {
...
 
-  formatCountryName(country) {
+  formatCountryName = (country) => {
     const lowercaseName = country.toLowerCase();
     const formattedName = lowercaseName.replaceAll(' ', '+');
 
     return formattedName;
   },
+
+  const checkForErr = (response) => {
```
Comment (pcmueller) [inline src/utilities/utils.js:20]: Nice, good call haven't different error handling for different status codes.

File: src/utilities/utils.js [hunk 2]
```diff
@@ -2,20 +2,32 @@ const utils = {
...
+      if(response.status >= 500) {
+        return '500: Uhoh, the angry computer gnome just drop kicked one of our servers. This page is temporarily unavailable.'
```
Comment (bhohnco) [inline src/utilities/utils.js:22]: This is awesome! love it

File: cypress/integration/Main_View_spec.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+describe('Show main view of Worm Hole App', () => {
```
Comment (pcmueller) [inline cypress/integration/Main_View_spec.js:1]: Very thorough UI testing - appreciate all your work on this!

**Review by bhohnco — APPROVED** [review]

**Review by pcmueller — COMMENTED** [review]
Really great foundation for our testing.

File: src/utilities/utils.js [hunk 4]
```diff
@@ -2,20 +2,32 @@ const utils = {
...
+      } else if(response.status === 404) {
+        return '404: oh no! Nothing in here. We cannot seem to find the page you are looking for.'
+      } else if (!response.ok) {
+        return 'Something small went wrong. Please try again later.'
```
Comment (bhohnco) [inline src/utilities/utils.js:26]: We'll need to test these error messages, and see what comes back.

**Review by bhohnco — APPROVED** [review]

**Pull request merged**